opus/application/static_media/js/mutationObserver.js

```diff
@@ -190,8 +201,9 @@ var o_mutationObserver = {
         let metadataSelector = $("#metadataSelector")[0];
         let metadataSelectorContents = $("#metadataSelectorContents")[0];
         let browseDialogModal = $("#galleryView.modal")[0];
-        let galleryView = $(".op-gallery-view")[0];
+        let galleryView = $(".gallery")[0];
```

Comment:
consider using $(".gallery").first(), etc

Reply:
I think we need to keep using $(".gallery")[0] because we have to pass in a Node to mutationObserver. If we use $(".gallery").first(), it will complain about the passed in $(".gallery").first() is not of type Node.

Reply:
i wondered that; thanks for explaination.